src/common/user-store.ts

```diff
@@ -28,6 +28,7 @@ export interface UserPreferences {
   downloadBinariesPath?: string;
   kubectlBinariesPath?: string;
   openAtLogin?: boolean;
+  storeTableConfig?: Object;
```

Comment:
Using `Object` is a bad idea, and doesn't help with typing in TS.

```suggestion
  storeTableConfig?: Record<string, Record<string, number>>;
```